policyengine_us/variables/gov/states/ne/tax/income/deductions/ne_standard_deduction.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class ne_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "NE standard deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://revenue.nebraska.gov/files/doc/tax-forms/2021/f_1040n_booklet.pdf"
+        "https://revenue.nebraska.gov/files/doc/2022_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_5.pdf"
+    )
+    defined_for = StateCode.NE
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        p = parameters(period).gov.states.ne.tax.income.deductions.standard
+        # count extra amounts for adults being elderly and/or blind
+        # number of head extras
+        head_extras = 0
```

Comment:
You can use `tax_unit("aged_blind_count", period)` here, as the [legal code](https://nebraskalegislature.gov/laws/statutes.php?statute=77-2716.01) points to the federal definition.